commcare_connect/opportunity/models.py

```diff
@@ -146,8 +146,19 @@ def number_of_users(self):
 
     @property
     def allotted_visits(self):
-        payment_units = self.paymentunit_set.all()
-        return sum([pu.max_total or 0 for pu in payment_units]) * self.number_of_users
+        return self.max_visits_per_user_new * self.number_of_users
+
+    @property
+    def max_visits_per_user_new(self):
```

Comment:
nit: `new` is not a great name for these since it doesn't explain why this is different (and if we add another one, will become out of date). ideally we would use a descriptive name, or delete the old column.